controllers/constants.go

```diff
@@ -64,13 +64,15 @@ const (
 	confTwillSecurityWorkerSecretDiskName = "twill.security.worker.secret.disk.name"
 	confTwillSecurityWorkerSecretDiskPath = "twill.security.worker.secret.disk.path"
 	confJMXServerPort                     = "jmx.metrics.collector.server.port"
+	confDefaultModeSecret                 = "default.mode.secret"
```

Comment:
nit: what about naming it secretDefaultMode and secret.default.mode instead

Reply:
Renamed - secret.mount.default.mode